docker/Dockerfile

```diff
@@ -159,6 +163,7 @@ RUN addgroup -g ${GID} ${USER} && adduser -u ${UID} -D -G ${USER} -g "" ${USER}
     if [ "$ENABLE_JNI" = "ON" ]; then apk add --no-cache openjdk8-jre-base; fi && \
     if [ "$ENABLE_PCAP" = "ON" ]; then apk add --no-cache libpcap; fi && \
     if [ "$ENABLE_USB_CAMERA" = "ON" ]; then apk add --no-cache libpng libusb; fi && \
+    if [ "$ENABLE_LUA_SCRIPTING" = "ON" ]; then apk add --no-cache lua; fi && \
```

Comment:
I think we should also add a check for ENABLE_SCRIPTING flag here. Also just a thought but maybe we could also include the ENABLE_ALL check here as well for all of these optional packages.

Reply:
I added the ENABLE_SCRIPTING check in https://github.com/apache/nifi-minifi-cpp/pull/1244/commits/c222bf84612ca4ff4bd85c6cf5dc79b03cbc1fed
i would skip the ENABLE_ALL for now because it is not that consistent (e.g. its not even considered when building the minifi-scripting library https://github.com/apache/nifi-minifi-cpp/blob/main/extensions/script/CMakeLists.txt#L47 ) 
I think we should refactor this whole ENABLE_ALL flag in a different ticket

Reply:
That's a good point, I agree.